codebuild/bin/install_openssl_3_0.sh

```diff
@@ -53,10 +70,23 @@ make -j $JOBS test
 make -j $JOBS install
 
 popd
+pushd $INSTALL_DIR
 
 # sym-link lib -> lib64 since codebuild assumes /lib path
-pushd $INSTALL_DIR
 ln -s lib64 lib
+
+# Openssl3 uses the openssl config file to enable fips
+# See https://docs.openssl.org/master/man7/fips_module/#making-all-applications-use-the-fips-module-by-default
+if $FIPS; then
+    config_dir=$(LD_LIBRARY_PATH=lib ./bin/openssl version -d | sed -r "s/OPENSSLDIR: \"(.*?)\"/\1/")
+    config="$config_dir"/openssl.cnf
```

Comment:
Don't we know where OPENSSLDIR is going to be based on the prefix we give `./Configure`? If not, this seems fine. But hardcoding might be simpler and reduce the risk of accidentally updating the system openssl config.

Reply:
We can guess that OPENSSLDIR is $INSTALL_DIR/ssl, and I doubt it would ever change, but `version -d` is the correct way. We can't accidentally update the system config, because we're not just calling `openssl`, we're calling `./bin/openssl` from $INSTALL_DIR.

But since I doubt the location will ever change, I just hardcoded it. Hopefully that causes less confusion.